src/os/bluestore/BlueStore.cc

```diff
@@ -4573,6 +4573,13 @@ BlueStore::BlueStore(CephContext *cct,
   _init_logger();
   cct->_conf.add_observer(this);
   set_cache_shards(1);
+  codel = new BlueStoreSlowFastCoDel(cct,
```

Comment:
Currently, we're checking for activated in a few places in BlueStoreSlowFastCodel.cc, but the timer etc are still running even when inactivated.  I'd rather just avoid instantiating codel at all while it's disabled and check for null in BlueStore before accessing it.  That would make it easier for developers to prove whether it was enabled when debugging problems.